app/utils/WkConf.scala

```diff
@@ -64,6 +64,13 @@ class WkConf @Inject()(configuration: Configuration) extends ConfigReader with L
     val children = List(User, Tasks, Cache, SampleOrganization)
   }
 
+  object SingleSignOn {
+    object OIDC {
+      val providerUrl: String = get[String]("singleSignOn.OIDC.providerURL")
+      val clientId: String = get[String]("singleSignOn.OIDC.clientID")
+    }
```

Comment:
```suggestion
    object OpenIdConnect {
      val providerUrl: String = get[String]("singleSignOn.openIdConnect.providerUrl")
      val clientId: String = get[String]("singleSignOn.openIdConnect.clientId")
    }
```
I’d prefer to avoid abbreviations and all-uppercase URL/ID also in the config :)